revenuecat_examples/purchase_tester/android/app/build.gradle

```diff
@@ -34,7 +34,7 @@ android {
     defaultConfig {
         applicationId "com.revenuecat.purchases_sample"
         minSdkVersion 16
-        targetSdkVersion 31
+        targetSdkVersion 30
```

Comment:
Target SDK version of 31 was causing this issue...

```
android:exported needs to be explicitly specified for element <activity#androidx.test.core.app.InstrumentationActivityInvoker$BootstrapActivity>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.
/Users/joshholtz/Developer/purchases-flutter/revenuecat_examples/purchase_tester/build/app/intermediates/tmp/manifest/androidTest/debug/tempFile1ProcessTestManifest3235334484840798035.xml:34:9-40:20
```

This target version didn't seem _super_ critical since this is the example app but still very weird and probably needs to be address in a better way for then the target version does need to be 31 again? 🤔

Reply:
Hmm weird. I just tried this branch and setting this version back to 31 and was able to build the android app without issues... Were you running through Android studio or something else?

Reply:
I was trying through command line with `./gradlew` but this was also failing on CI for me 😬

Reply:
I still can't really figure out what is causing this and another fix 😮‍💨 

@tonidero @vegaro Is this okay to ship in alpha and beta?

Reply:
I was taking a look and was able to repro when building integration tests, not the app itself. And I was taking a look at the cause. Looks like we increased the gradle and gradle plugin versions since the last release. My bet would be that broke this. Considering this is only in the purchase tester app. I think it would be ok to do this for alpha/beta, but we should fix it before final release to confirm there are no issues. My guess it should be updating some dependencies